### PR TITLE
Fix #3530: Merge home fragments

### LINKED ISSUE #3530: Rename concept_card dimens properly
In dimens.xml file, Rename all the dimens of concept card fragment, according to proper convention like
concept_card_explanation_text_marginEnd(wrong convention) -> concept_card_explanation_text_margin_end(right convention)

Note: As you change the dimens.xml file, accordingly also change the concept_card XML file

### PR DESCRIPTION
**Pull request by bkaur-bkj**
## Explanation

Fixes #6447
Merged the 4 home-fragment xmls (landscape/portrait of phone and tablet )into one home- fragment  
Have added screenshots of home-fragment before and after changes in android phone. Sorry could not attach screenshots for tablet as emulator was not being supported by my studio.
**Before changes** 

WhatsApp Image 2021-08-25 at 9 27 53 PM (1)

WhatsApp Image 2021-08-25 at 9 27 53 PM

**After changes** 

WhatsApp Image 2021-08-25 at 9 29 56 PM

WhatsApp Image 2021-08-25 at 9 29 52 PM

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by bkaur-bkj** [comment]
@rt4914 pls take a look

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <component name="RunConfigurationProducerService">
     <option name="ignoredProducers">
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
```
Comment (rt4914) [inline .idea/runConfigurations.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: app/src/main/res/values-land/dimens.xml [hunk 3]
```diff
@@ -276,4 +276,7 @@
   <dimen name="app_version_image_view_margin_start">36dp</dimen>
   <dimen name="app_version_text_view_margin_end">36dp</dimen>
 
+
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Remove this extra blank line

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 4]
```diff
@@ -301,4 +301,9 @@
   <dimen name="app_version_image_view_margin_start">24dp</dimen>
   <dimen name="app_version_text_view_margin_end">28dp</dimen>
 
+  <dimen name="home_fragment_padding_top">48dp</dimen>
+  <dimen name="home_fragment_padding_bottom">100dp</dimen>
+
+
+
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: Remove these 2 extra blank line. Keep only one blank line

File: app/src/main/res/values-sw600dp-port/dimens.xml [hunk 5]
```diff
@@ -303,4 +303,6 @@
   <dimen name="app_version_image_view_margin_start">24dp</dimen>
   <dimen name="app_version_text_view_margin_end">28dp</dimen>
 
+  <dimen name="home_fragment_padding_top">48dp</dimen>
+  <dimen name="home_fragment_padding_bottom">100dp</dimen>
 </resources>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-port/dimens.xml:309]: Add one blank line above this.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@bkaur-bkj PTAL

File: .idea/misc.xml (hunk 1 above)
Comment (bkaur-bkj) [reply .idea/misc.xml]: Done.

File: .idea/runConfigurations.xml (hunk 2 above)
Comment (bkaur-bkj) [reply .idea/runConfigurations.xml]: done

File: app/src/main/res/values-land/dimens.xml (hunk 3 above)
Comment (bkaur-bkj) [reply app/src/main/res/values-land/dimens.xml]: done

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 4 above)
Comment (bkaur-bkj) [reply app/src/main/res/values-sw600dp-land/dimens.xml]: done

File: app/src/main/res/values-sw600dp-port/dimens.xml (hunk 5 above)
Comment (bkaur-bkj) [reply app/src/main/res/values-sw600dp-port/dimens.xml:309]: done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**